scripts/generate-railroad-diagram.mts

```diff
@@ -10,7 +10,7 @@
  */
 
 import { PliParserInstance } from '../packages/language/src/parser/parser';
-import { createSyntaxDiagramsCode } from 'chevrotain';
+import { createSyntaxDiagramsCode } from '../packages/language/node_modules/chevrotain/lib/src/api.js';
```

Comment:
Suggestion: IMO we should add `chevrotain` as a dev dependency to the root project.